### ISSUE TITLE
No separator line between my nodes and sverchok-extra

### ISSUE THREAD

**Issue by wassimj** [portnov/sverchok-extra#34]
I have been modelling my Topologic nodes on sverchok extra and I just noticed that in the Shift-A menu, there is no separation between the sverchok-extra nodes and the Topologic nodes. I am suspecting somehow they are being read into the same category? If so, where should I look to make sure there is a separator line between the two packages? Thank you
![Screenshot from 2021-02-11 11-20-27](https://user-images.githubusercontent.com/6302909/107630264-2eec6480-6c5b-11eb-8948-111144557f92.png)

**Comment by vicdoval** [comment]
@wassimj  Don't you think it would be cleaner to have all the nodes inside a Topologic menu with submenus?

**Comment by wassimj** [comment]
@vicdoval Yes 100% and I had it like that, except I don’t know how to create sub-sub-menus. I would like to create a structure like:

Topologic
——Vertex
————ByCoordinates
————Coordinates
——Edge
————ByStartVertexEndVertex
————StartVertex
————EndVertex

**Comment by vicdoval** [comment]
The basic idea is creating your own Topologic menu with the categories you desire and then you plug it to the regular menu.
In Blender every menu, or sub-menu has to be created and registered independently.
In Sverchok this is automated so you just need to edit the index.md file and the rest is made under the hood.
In sv-extra there is not a index.md file so it is a little different which shows there is many ways to code this task.

So you can try to make it your own way or maybe if you have a public repository with your code maybe some else can help you (maybe me but it has to be Windows ready at the moment).

**Comment by satabol** [comment]
Hi @wassimj

Sverchok-Extra is now updated and one can create sub-sub menu now. See https://github.com/portnov/sverchok-extra/blob/master/nodes_index.py for example:
![image](https://github.com/portnov/sverchok-extra/assets/14288520/23d4aa07-0b20-4752-aa3d-34663c444958)


If your addon is another project then you need update functions to build your menu in the file https://github.com/portnov/sverchok-extra/blob/master/dependencies.py and see any class in the Extra to see the difference with previous node creation pipeline (not many changes):

![image](https://github.com/portnov/sverchok-extra/assets/14288520/c0711ad7-22f4-4a3a-8bd6-b495836025c6)
